quic/s2n-quic-transport/src/path/manager.rs

```diff
@@ -332,6 +332,17 @@ impl<Config: endpoint::Config> Manager<Config> {
         let local_address = path_handle.local_address();
         let active_local_addr = self.active_path().local_address();
         let active_remote_addr = self.active_path().remote_address();
+        // The peer has intentionally tried to migrate to a new path because they changed
+        // their destination_connection_id. This is considered an "active" migration.
+        let active_migration =
+            self.active_path().local_connection_id != datagram.destination_connection_id;
```

Comment:
wondering if we can also link to a citation here since this is not explicitly stated in the RFC

Reply:
I took that more as the reason why we are using a new CID for sending back to the peer, so more relevant to where it is currently